lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -1,16 +1,28 @@
+require "pathname"
+
 module CC
   module Analyzer
     class IssueRelativePathValidation < Validation
       def valid?
-        path && !path.start_with?("/")
+        path &&
+          !path.start_with?("/") &&
+          File.exist?(path) &&
```

Comment:
`Pathname#realpath` touches the filesystem and will error if the file doesn't exist.

```
Errno::ENOENT:
       No such file or directory @ realpath_rec - /private/tmp/source.rb
```

I could rescue that exception in `#relative_to?` instead. What do you think?